myst_nb/transform.py

```diff
@@ -47,11 +51,11 @@ def apply(self):
             outputs = cell.get("outputs", [])
             if node.get("inline", False):
                 output_nodes = cell_output_to_nodes_inline(
-                    outputs, RENDER_PRIORITY[builder], True, output_dir, None
+                    outputs, RENDER_PRIORITY.get(builder, 'html'), True, output_dir, None
```

Comment:
Please also add a sphinx warning towards the start of this transform; to indicate that no render priority is available for the given builder

Reply:
can you point me to what a sphinx warning looks like?

Reply:
https://www.sphinx-doc.org/en/master/extdev/logging.html#sphinx.util.logging.SphinxLoggerAdapter.warning